app/helpers/payouts_helper.rb

```diff
@@ -156,6 +230,213 @@ def displayable_payout_period_range(previous_payment, payout_period_end_date)
     end
   end
 
+  def generate_multi_payment_success_message(completed_payments, payment)
+    total_amount = completed_payments.sum do |info|
+      amount = info["amount_cents"]
+      case amount
+      when Integer
+        amount
+      when String
+        amount.to_i
+      when nil
+        0
+      else
+        0
+      end
+    end
+
+    return nil if total_amount == 0
+
+    completed_date = completed_payments.first&.dig("completed_at") || payment.created_at
+    formatted_date = formatted_payout_date(completed_date)
+
+    "This payout was paid in multiple payments: **#{formatted_dollar_amount(total_amount)}** was paid via PayPal on **#{formatted_date}**."
+  end
+
+  def calculate_partial_payout_info(payment)
+    return {} unless payment.was_created_in_split_mode? && payment.split_payments_info.present?
```

Comment:
The partially failed error message in the design is not related to split PayPal payouts actually, but it is for when a part of the amount is already paid out directly to the connected PayPal account of the seller. Please see `User::Stats::paypal_payout_net_cents` for reference or see how the "PayPal Payouts" head in the payouts table is calculated.